doc/build-freebsd.md

```diff
@@ -9,16 +9,16 @@ This guide describes how to build bitcoind, command-line utilities, and GUI on F
 ### 1. Install Required Dependencies
 Run the following as root to install the base dependencies for building.
 
-```bash
```

Comment:
Is there any  reason to change these, aren't`bash` and `sh` an alias of the same thing? At the same time, they are still missing in certain places, i.e line 44.

Reply:
Thanks! Updated.

Reply:
> aren't `bash` and `sh` an alias of the same thing?

No, those are two different shells. On some (all?) Linux distros it happens that only `bash` is installed and `sh` is linked to it, but this is not the case on FreeBSD. Same with `vi` and `vim`.

Anyway, the commands used in this doc are simple and would work with both `bash` and `sh`.

Reply:
> No, those are two different shells.

I know, but in markdown parsing they are the same thing.